mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -14,24 +14,33 @@
 import xarray as xr
 import datetime
 from netCDF4 import Dataset as netcdf_dataset
-import os.path
 
-from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
+from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, \
+    remove_repeated_time_index
 from ..shared.plot.plotting import plot_global_comparison
 from ..shared.interpolation.interpolate import interp_fields, init_tree
 from ..shared.constants import constants
 
 from ..shared.io import StreamsFile
 
-def ocn_modelvsobs(config, field):
+
+def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
 
     """
     Plots a comparison of ACME/MPAS output to SST or MLD observations
 
+    If present, streamMap is a dictionary of MPAS-O stream names that map to
+    their mpas_analysis counterparts.
+
+    If present, variableMap is a dictionary of MPAS-O variable names that map
+    to their mpas_analysis counterparts.
+
     Authors: Luke Van Roekel, Milena Veneziani, Xylar Asay-Davis
-    Modified: 10/27/2016
+    Modified: 12/07/2016
     """
 
+    field = field.lower()
```

Comment:
> I'm wondering if we shouldn't keep all PEP8 changes in their own PRs for simplicity in the future. It is pretty hard to review the code with substantive changes as well as PEP8 changes because it is not always clear what changes correspond to what.

I'm sympathetic to this.  It's hard for me to work on code effectively at this point that isn't PEP8 compliant because spyder gives me a ton of warnings that I usually use to find bugs in my code.  At the same time, I don't really have time to go through the repo and make everything PEP8 compliant first, as a separate PR.  So I fix things up as I go along.

A compromise approach for the future might be for me to do a separate commit in a given PR for the PEP8 changes and for the substantive changes.  I usually try to do it that way but I combined the two in this PR and I'm sorry for doing that.

I'm very concerned that too many PRs will make this work untenable.  As it is, it is taking a surprisingly long time in general to get PRs reviewed in this repository.  (I generally try to do reviews within a day or two of receiving them, and I leave them unreviewed only if there are significant changes for the submitter to make before I can proceed.)  So I am also concerned that breaking PRs into more PRs will not make things move any faster.